plugins/web/opentelemetry-plugin-user-interaction/src/userInteraction.ts

```diff
@@ -227,6 +227,19 @@ export class UserInteractionPlugin extends BasePlugin<unknown> {
     return patched;
   }
 
+  // utility method to deal with the Function|EventListener nature of addEventListener
+  private _invokeListener(
+    listener: Function | EventListenerObject,
+    target: any,
```

Comment:
Shouldn't these `any` be `unknown`?

Reply:
I was just copying through the types from the parent function (`addEventListenerPatched`) - just trying to keep the PR small rather than revisit types all through the file.  Happy to expand scope or file a separate PR if you'd like to go that way.